### ISSUE TITLE
You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'IF' at line 1

### ISSUE THREAD

**Issue by thecancerus** [rankmath/seo-by-rank-math#35]
**Describe the bug**
I installed the Rank Math V 1.0.53 plugin on a website with WordPress 5.5.3, PHP 7.3 & MariaDB 10.3.25, and when I tried to activate it gave the following error

_You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'IF' at line 1_

**Additional context**
I tried to debug the issue, where is what I found, dbDelta when tries to parse the create query it results in picking up the wrong table name. As shown in the dump of ```$cqueries``` variable in dbDelta function.

 array(1) 
[
    "IF" => string(496) "CREATE TABLE IF NOT EXISTS wp_rank_math_404_logs (
				id BIGINT(20) unsigned NOT NULL AUTO_INCREMENT,
				uri VARCHAR(255) NOT NULL,
				accessed DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
				times_accessed BIGINT(20) unsigned NOT NULL DEFAULT 1,
				ip VARCHAR(50) NOT NULL DEFAULT '',
				referer VARCHAR(255) NOT NULL DEFAULT '',
				user_agent VARCHAR(255) NOT NULL DEFAULT '',
				PRIMARY KEY (id),
				KEY uri (uri(191))
			) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci"
]

So when dbDelta tries to run the following line it results in the above error.

```$tablefields = $wpdb->get_results( "DESCRIBE {$table};" );```

basically, it tries to execute the following query 
```DESCRIBE IF;```

**Comment by surajv** [comment]
Hello @thecancerus,

Could you please confirm from **Rank Math >> Staus & Tools >> System Status >> Rank Math Toggle**, if all the required tables exist or not?

Thank you.

**Comment by thecancerus** [comment]
@surajv the moment I activated the plugin, I would get the fatal error and the site won't load, so i ended up changing the sql query in rankmath plugin to fix the setup for me.

**Comment by surajv** [comment]
@thecancerus can you please share the final query after your edits?

Thank you.

**Comment by thecancerus** [comment]
I had reverted the file back to original once table got created. Here is what is had changed 
Original
`CREATE TABLE {$wpdb->prefix}rank_math_404_logs (
				id BIGINT(20) unsigned NOT NULL AUTO_INCREMENT,
				uri VARCHAR(255) NOT NULL,
				accessed DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
				times_accessed BIGINT(20) unsigned NOT NULL DEFAULT 1,
				ip VARCHAR(50) NOT NULL DEFAULT '',
				referer VARCHAR(255) NOT NULL DEFAULT '',
				user_agent VARCHAR(255) NOT NULL DEFAULT '',
				PRIMARY KEY (id),
				KEY uri (uri(191))
			) $collate;`
Mine

`CREATE TABLE IF NOT EXISTS {$wpdb->prefix}rank_math_404_logs (
				id BIGINT(20) unsigned NOT NULL AUTO_INCREMENT,
				uri VARCHAR(255) NOT NULL,
				accessed DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
				times_accessed BIGINT(20) unsigned NOT NULL DEFAULT 1,
				ip VARCHAR(50) NOT NULL DEFAULT '',
				referer VARCHAR(255) NOT NULL DEFAULT '',
				user_agent VARCHAR(255) NOT NULL DEFAULT '',
				PRIMARY KEY (id),
				KEY uri (uri(191))
			) $collate;`

**Comment by thecancerus** [comment]
Hey guys, I spent some time on this. you should remove the  "IF NOT EXISTS" from the query as the dbDelta function in WordPress checks for table before executing the query.

In my case, the Awesome Enterprise plugin was blocking the execution in case of error hence i came to know about it, while normally WordPress silently let it go.
